[FR] Add `Beatmaps Nominators/General Information` (#5569)
ppy/osu-wiki: wiki/People/The_Team/Beatmap_Nominators/General_Information/fr.md

# Informations générales pour les Beatmap Nominators

> Êtes vous un [**Beatmap Nominator**](/wiki/People/The_Team/Beatmap_Nominators) *(BN)*, ou  voulez-vous le devenir ? Si la réponse est oui, vous êtes au bon endroit ! Cet article contient toutes les informations globales que vous aurez besoin en tant que nouveau Beatmap Nominator.

Reviewer: ```suggestion
Êtes vous un [**Beatmap Nominator**](/wiki/People/The_Team/Beatmap_Nominators) *(BN)*, ou voulez-vous en devenir un ? Si la réponse est oui, vous êtes au bon endroit ! Cet article contient toutes les informations globales que vous aurez besoin en tant que nouveau Beatmap Nominator.
```

> Êtes vous un [**Beatmap Nominator**](/wiki/People/The_Team/Beatmap_Nominators) *(BN)*, ou voulez-vous en devenir un ? Si la réponse est oui, vous êtes au bon endroit ! Cet article contient toutes les informations globales que vous aurez besoin en tant que nouveau Beatmap Nominator.

Reviewer 2: ```suggestion
Êtes vous un [**Beatmap Nominator**](/wiki/People/The_Team/Beatmap_Nominators) *(BN)*, ou voulez-vous en devenir un ? Si la réponse est oui, vous êtes au bon endroit ! Cet article contient toutes les informations que vous aurez besoin en tant que nouveau Beatmap Nominator.
```

> ## Liens Importants

Reviewer 2: ```suggestion
## Liens importants
```

> - [Règles des Beatmaps Nominators](/wiki/People/The_Team/Beatmap_Nominators/Rules) : Explique à la fois comment entreprendre diverses actions, ainsi comment ne pas faire les choses - vous voudrez aussi éviter de les casser dans toutes les situations, donc soyez sûr de les lire attentivement ! Vous pourrez toujours demander à un membre de la [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) *(NAT)* si vous n'êtes pas sûr à propos de quelque chose et ils se feront une joie de vous aider.

Reviewer: ```suggestion
- Les [règles des Beatmaps Nominators](/wiki/People/The_Team/Beatmap_Nominators/Rules) : Explique à la fois comment entreprendre diverses actions, ainsi que comment ne pas faire les choses - il vous est aussi préférable d'éviter de les briser en toutes situations, donc soyez sûr de les avoir lu attentivement ! Vous pourrez toujours demander à un membre de la [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) *(NAT)* si vous n'êtes pas sûr à propos de quelque chose et ils se feront une joie de vous aider.
```

> - Les [règles des Beatmaps Nominators](/wiki/People/The_Team/Beatmap_Nominators/Rules) : Explique à la fois comment entreprendre diverses actions, ainsi que comment ne pas faire les choses - il vous est aussi préférable d'éviter de les briser en toutes situations, donc soyez sûr de les avoir lu attentivement ! Vous pourrez toujours demander à un membre de la [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) *(NAT)* si vous n'êtes pas sûr à propos de quelque chose et ils se feront une joie de vous aider.

Reviewer 2: ```suggestion
- Les [règles des Beatmaps Nominators](/wiki/People/The_Team/Beatmap_Nominators/Rules) : Explique à la fois comment entreprendre diverses actions, ainsi que comment ne pas faire les choses - il vous est aussi préférable d'éviter de les briser en toutes situations, donc soyez sûr de les avoir lu attentivement ! Vous pourrez toujours demander à un membre de la [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) *(NAT)* si vous n'êtes pas sûr à propos de quelque chose, ils se feront une joie de vous aider.
```

`Explique à la fois comment entreprendre diverses actions, ainsi que comment ne pas faire les choses` C'est bizarre je trouve comme phrase, faut voir comment la reformuler

Reviewer: Ouai j'ai écrit que la fin mais je suis pas satisfait même si théoriquement ça suit bien la version originale

Reviewer: Peut-être ça ça serait mieux
```suggestion
- Les [règles des Beatmaps Nominators](/wiki/People/The_Team/Beatmap_Nominators/Rules) : Explique en détails à la fois ce qu'il faut faire, et ce qu'il ne faut pas faire - il vous est aussi préférable d'éviter de les briser en toutes situations, donc soyez sûr de les avoir lu attentivement ! Vous pourrez toujours demander à un membre de la [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) *(NAT)* si vous n'êtes pas sûr à propos de quelque chose et ils se feront une joie de vous aider.
```

Reviewer 2: Déjà mieux

> - [Code de conduite pour le Mapping et le Modding](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping) : En tant que BN, vous êtes censés être une figure exemplaire pour la communauté, quelqu'un que les autres admirent et respectent, donc une faute peut vous conduire à une réprimande ou peut vous expulser.

Reviewer: ```suggestion
- Le [code de conduite pour le Mapping et le Modding](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping) : En tant que BN, vous êtes censés être une figure exemplaire pour la communauté, quelqu'un que les autres admirent et respectent, donc une faute peut vous conduire à une réprimande ou vous faire expulser.
```

> - Le [code de conduite pour le Mapping et le Modding](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping) : En tant que BN, vous êtes censés être une figure exemplaire pour la communauté, quelqu'un que les autres admirent et respectent, donc une faute peut vous conduire à une réprimande ou vous faire expulser.

Reviewer 2: ```suggestion
- Le [code de conduite pour le Mapping et le Modding](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping) : En tant que BN, vous êtes censés être une figure exemplaire pour la communauté, quelqu'un que les autres admireront et respecterons, donc une faute peut vous conduire à un avertissement, ou pire, vous pouvez vous faire expulser.
```

Reviewer: Alors d'accord pour mettre au futur mais faut mettre un "t" pour les deux à la fin, "avertissement" et "ou pire," je suis d'accord aussi par contre le "vous pouvez" on peut s'en passer imo

> - [Critères de Classement](/wiki/Ranking_Criteria) : Si vous êtes déjà un BN, alors vous devriez déjà être familier avec ceci. Essentiellement, il sert à expliquer quels aspects du mapping sont ou ne sont pas autorisés dans la section des maps classées, ainsi qu'à inclure quelques directives générales que les gens devraient suivre dans la plupart des cas. 

Reviewer: J'aimerai avoir la confirmation de @DoryanR pour savoir comment appeler la "Ranked section" parce que ta traduction risque de ne pas être uniforme avec des anciennes traductions.
```suggestion
- Les [critères de Classement](/wiki/Ranking_Criteria) : Si vous êtes déjà un BN, alors vous devriez déjà être familier avec ceci. Essentiellement, il sert à expliquer quels aspects du mapping sont ou ne sont pas autorisés dans la Ranked section, ainsi qu'à inclure quelques directives générales que les gens doivent suivre dans la plupart des cas. 
```

> - Les [critères de Classement](/wiki/Ranking_Criteria) : Si vous êtes déjà un BN, alors vous devriez déjà être familier avec ceci. Essentiellement, il sert à expliquer quels aspects du mapping sont ou ne sont pas autorisés dans la Ranked section, ainsi qu'à inclure quelques directives générales que les gens doivent suivre dans la plupart des cas. 

Reviewer 2: ```suggestion
- Les [critères de classement](/wiki/Ranking_Criteria) : Si vous êtes déjà un BN, alors vous devriez déjà être familier avec ceci. Essentiellement, il sert à expliquer quels aspects du mapping sont ou ne sont pas autorisés dans la catégorie classée, ainsi qu'à inclure quelques directives générales que les gens doivent suivre dans la plupart des cas. 
```

> - [Informations sur le Veto pour une Beatmap](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) : Pour les beatmaps avec des problèmes subjectifs de qualité qui ne violent pas nécessairement les règles ou les directives de classement, mais qui, selon vous, rendent la beatmap inéligible à la Ranked section, le veto de beatmaps permet de retirer la qualification d'une beatmap.

Reviewer 2: ```suggestion
- [Informations sur le Veto pour une beatmap](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) : Pour les beatmaps avec des problèmes subjectifs de qualité qui ne violent pas nécessairement les règles ou les directives de classement, mais qui, selon vous, rendent la beatmap inéligible à la catégorie classée, le veto d'une beatmap permet de retirer la qualification d'une beatmap.
```

> - Le [Mapset Verifier](https://osu.ppy.sh/community/forums/topics/943895): Un outil créé par ([Naxess](https://osu.ppy.sh/users/8129817)), un membre du NAT . Cet outil permet de vérifier un large éventail de problèmes rendant une beatmap non-classable, ainsi qu'une fonctionnalité de capture instantanée pour facilement comparer les versions d'une map, ce qui est utile lors des revérifications. **Cela ne doit pas remplacer la vérification d'une beatmap par vous même**, mais plutôt être utilisé comme un outil de double-vérification au cas-où vous auriez loupé quelque chose.

Reviewer 2: ```suggestion
- Le [Mapset Verifier](https://osu.ppy.sh/community/forums/topics/943895) : Un outil créé par ([Naxess](https://osu.ppy.sh/users/8129817)), un membre du NAT . Cet outil permet de vérifier un large éventail de problèmes rendant une beatmap non-classable, ainsi qu'une fonctionnalité de capture instantanée pour facilement comparer les versions d'une map, ce qui est utile lors des revérifications. **Cela ne doit pas remplacer la vérification d'une beatmap par vous même**, mais plutôt être utilisé comme un outil de double-vérification au cas-où vous auriez loupé quelque chose.
```

> ## Les Capacités des Beatmap Nominator

Reviewer 2: ```suggestion
## Les capacités des Beatmap Nominators
```

Les nominations et les resets de nominations démontrent les efforts d'un BN lors de la vérifications des maps, indiquant la progression d'une map à travers le processus de classement. En tant que BN vous avez accès à ces outils et vous devez les utiliser à bon escient.

### Nominer

> ![nominate button](img/nominate.png)

Reviewer 2: ```suggestion
![bouton de nomination](img/nominate.png)
```

> **Ce bouton vous permet de nominer des maps.** Pour nominer une beatmap, le mapset doit avoir au moins 5 Hype et aucun Problem ou Suggestion stamps non-résolu. Assurez vous que vous êtes satisfait de toutes les difficultés, de même pour les autres modes de jeu, avant d'appuyer ici. Si la beatmap a fait l'objet d'un veto par le passé, vous devez faire attention à ce que le veto soit retiré avant de faire avancer un mapset. Quand une beatmap possède 2 nominations, elle sera qualifiée et entrera dans la file d'attente pour le classement.

Reviewer 2: ```suggestion
**Ce bouton vous permet de nominer des maps.** Pour nominer une beatmap, cette dernière doit avoir au moins 5 Hype et aucun problèmes ou de suggestions non résolu. Assurez vous que vous êtes satisfait de toutes les difficultés, de même pour les autres modes de jeu, avant d'utiliser ce bouton. Si la beatmap a fait l'objet d'un veto par le passé, vous devez faire attention à ce que le veto soit retiré avant de faire avancer une beatmap. Quand une beatmap possède 2 nominations, elle sera qualifiée et entrera dans la file d'attente pour le classement.
```

### Disqualifier ou réinitialiser la nomination

> ![disqualify and reset buttons](img/reset_dq.png)

Reviewer 2: ```suggestion
![boutons de disqualification et de réinitialisation](img/reset_dq.png)
```

> 

Reviewer 2: Colle à la ligne 25 (l'image de la disqualification) ceci : `![Bouton de réinitialisation de la nomination](img/reset_q-FR.png "Merci à Zelq pour la photo")`

> **Cela vous permet de réinitialiser la progression d'une beatmap dans le processus de classement.** Si une beatmap est actuellement nominée ou qualifiée, vous en tant que nominateur avez la responsabilité de réinitialiser la nomination d'une beatmap en postant un problème en utilisant un Problem stamp. Si une beatmap est qualifiée, elle ne peut être disqualifiée que par un Full BN. Ces options sont généralement utilisées quand il y a un problème majeur ou inclassable qui doit être résolu avant d'envisager le classement.

Reviewer 2: ```suggestion
**Cela vous permet de réinitialiser la progression d'une beatmap dans le processus de classement.** Si une beatmap est actuellement nominée ou qualifiée, vous en tant que nominateur avez la responsabilité de réinitialiser la nomination d'une beatmap en postant un problème. Si une beatmap est qualifiée, elle ne peut être disqualifiée que par un BN non probatory. Ces options sont généralement utilisées quand il y a un problème majeur ou inclassable qui doit être résolu avant d'envisager le classement.
```

Étapes pour disqualifier une beatmap :

> - Assurez vous que vous êtes sur la discussion de la beatmap pour le bon set.

Reviewer 2: ```suggestion
- Assurez vous que vous êtes sur la discussion de la beatmap pour la bonne difficulté.
```

- Écrivez votre raison de disqualification dans le champ prévu à cet effet. Assurez vous que la première ligne résume la disqualification car c'est ce qui est affiché en haut de la page de discussion.
- Si la disqualification est liée à un rapport, associez également le(s) post(s) de discussion de beatmaps respectifs.

> - Appuyez sur le bouton "**Disqualifier**"  en dessous du champ de texte (celui que remplace le bouton "Problème").

Reviewer 2: Il est traduit, je l'ai vu passer sur le projet osu!web, sur [Crowdin](https://crowdin.com/translate/osu-web/all/en-fr?filter=basic&value=0#q=Disqualify)

> - Appuyez sur le bouton "**Disqualify**"  en dessous du champ de texte (celui qui remplace le bouton "Problème").

Reviewer 2: ```suggestion
- Appuyez sur le bouton "**Disqualifier**"  en dessous du champ de texte (celui qui remplace le bouton "Problème").
```

> En tant qu'extension à cela, vous pouvez aussi [veto une beatmap](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto), empêchant une beatmap d'être renominée jusqu'à ce que vous donniez une permission spéciale ou que le veto soit retiré par la médiation du veto. Cependant, placer un veto demande aussi que vous discutiez correctement du problème avec le mappeur et que vous essayiez de parvenir à une consensus. Discuter du problème avec le Nominateur original pourrait vous donner une nouvelle perspective sur la question, mais vous n'êtes pas obligé de le faire.

Reviewer: Je pense qu'il faut trouver autre chose pour "En tant qu'extension à cela" aussi
```suggestion
En tant qu'extension à cela, vous pouvez aussi [veto une beatmap](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto), empêchant une beatmap d'être renominée jusqu'à ce que vous donniez une permission spéciale ou que le veto soit retiré par la médiation du veto. Cependant, placer un veto demande aussi que vous discutiez correctement du problème avec le mappeur et que vous essayiez de parvenir à un consensus. Discuter du problème avec le Nominateur original pourrait vous donner un nouveau point de vue sur la question, mais vous n'êtes pas obligé de le faire.
```

> En tant qu'extension à cela, vous pouvez aussi [veto une beatmap](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto), empêchant une beatmap d'être renominée jusqu'à ce que vous donniez une permission spéciale ou que le veto soit retiré par la médiation du veto. Cependant, placer un veto demande aussi que vous discutiez correctement du problème avec le mappeur et que vous essayiez de parvenir à un consensus. Discuter du problème avec le Nominateur original pourrait vous donner un nouveau point de vue sur la question, mais vous n'êtes pas obligé de le faire.

Reviewer 2: ```suggestion
De plus, vous pouvez également ajouter un [veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) sur une beatmap, l'empêchant d'être renominée jusqu'à ce que vous donniez une permission spéciale ou que le veto soit retiré par la médiation du veto. Cependant, placer un veto demande aussi que vous discutiez correctement du problème avec le mappeur et que vous essayiez de parvenir à un consensus. Discuter du problème avec le nominateur original pourrait vous donner un nouveau point de vue sur la question, mais vous n'êtes pas obligé de le faire.
```

## Fonctions supplémentaires d'un Beatmap Nominator

En plus de pouvoir nominer des beatmaps ou de réinitialiser leurs nominations, les BN ont également d'autres fonctions.

> ### Médiation des Veto

Reviewer 2: ```suggestion
### Médiation des vetos
```

> Quand une médiation de veto est lancée, un jury choisi au hasard va être sélectionné. Si vous êtes l'un des membres choisi pour une médiation de veto, le NAT va vous envoyer un message avec des informations additionnelles sur comment utiliser le vote sur le [site BN](http://bn.mappersguild.com). Si vous ne pouvez pas participer, informez un membre du NAT afin que un autre BN puisse être choisi pour le jury à votre place.

Reviewer: ```suggestion
Quand une médiation de veto est lancée, un jury choisi au hasard va être sélectionné. Si vous êtes l'un des membres choisi pour une médiation de veto, le NAT va vous envoyer un message avec des informations additionnelles sur comment utiliser le vote sur le [site BN](http://bn.mappersguild.com). Si vous ne pouvez pas participer, informez un membre du NAT afin qu'un autre BN puisse être choisi pour le jury à votre place.
```

> Quand une médiation de veto est lancée, un jury choisi au hasard va être sélectionné. Si vous êtes l'un des membres choisi pour une médiation de veto, le NAT va vous envoyer un message avec des informations additionnelles sur comment utiliser le vote sur le [site BN](http://bn.mappersguild.com). Si vous ne pouvez pas participer, informez un membre du NAT afin qu'un autre BN puisse être choisi pour le jury à votre place.

Reviewer 2: ```suggestion
Quand une médiation de veto est lancée, un jury choisi au hasard va être sélectionné. Si vous êtes l'un des membres choisi pour une médiation de veto, un membre du NAT va vous envoyer un message avec des informations additionnelles sur comment utiliser le vote sur le [site des BN](http://bn.mappersguild.com). Si vous ne pouvez pas participer, informez un membre du NAT afin qu'un autre BN puisse être choisi pour le jury à votre place.
```

> ### Assurance de Qualité

Reviewer 2: ```suggestion
### Assurance qualité
```

> Les Full BN peuvent faire une Assurance de qualité en vérifiant à tout moment les beatmaps qualifiées pour détecter des problèmes. Le comité de l'assurance qualité peut être vu [sur le site BN](https://bn.mappersguild.com/qualityassurance).

Reviewer: ```suggestion
Les Full BN peuvent faire une Assurance de Qualité en vérifiant à tout moment les beatmaps qualifiées pour détecter des problèmes. Le comité de l'assurance qualité peut être vu [sur le site BN](https://bn.mappersguild.com/qualityassurance).
```

> Les Full BN peuvent faire une Assurance de Qualité en vérifiant à tout moment les beatmaps qualifiées pour détecter des problèmes. Le comité de l'assurance qualité peut être vu [sur le site BN](https://bn.mappersguild.com/qualityassurance).

Reviewer 2: ```suggestion
Les BN non probatory peuvent faire une "assurance qualité" en vérifiant à tout moment les beatmaps qualifiées pour détecter des problèmes. Le comité de l'assurance qualité peut être vu [sur le site des BN](https://bn.mappersguild.com/qualityassurance).
```

### Review du contenu

> Tous les BN peuvent contribuer à la modération des morceaux ou des images soit en y soumettant du contenu ou en y votant. Le processus de vote complet et la façon dont les votes sont comptés peuvent être lus sur l'[article sur le processus du vote du contenu](/wiki/Rules/Content_Voting_Process).

Reviewer: ```suggestion
Tous les BN peuvent contribuer à la modération des musiques ou des images soit en y soumettant du contenu à review ou en y votant. Le processus de vote complet et la façon dont les votes sont comptés peuvent être lus sur l'[article sur le processus du vote du contenu](/wiki/Rules/Content_Voting_Process).
```

> ### Évaluations Simulées

Reviewer 2: ```suggestion
### Évaluations simulées
```

> Les Full BN peuvent être assignés au hasard pour faire des évaluations simulées sur les requêtes des BN aux côtés des vrais du NAT. Ils peuvent choisir de ne pas y participer à tout moment, et il n'est pas nécessaire de faire des évaluations assignées. Les évaluations simulées sont principalement utilisées comme informations pour le NAT pour évaluer les futurs membres du NAT, bien que les détails présentés dans les évaluations simulées puissent être ajoutées à l'occasion aux commentaires des candidats ou servir de bris d'égalité si le NAT ne peut pas se décider par lui-même.

Reviewer: J'hésite à remettre tiebreaker en vrai.
```suggestion
Les Full BN peuvent être assignés au hasard pour faire des évaluations simulées sur les requêtes des BN aux côtés des vrais NAT. Ils peuvent choisir de ne pas y participer à tout moment, et il n'est pas nécessaire de faire des évaluations assignées. Les évaluations simulées sont principalement utilisées comme informations pour le NAT pour évaluer les futurs membres du NAT, bien que les détails présentés dans les évaluations simulées puissent être ajoutés à l'occasion aux commentaires des candidats ou bien pour éviter une égalité si le NAT ne peut pas se décider par lui-même.
```

> Les Full BN peuvent être assignés au hasard pour faire des évaluations simulées sur les requêtes des BN aux côtés des vrais du NAT. Ils peuvent choisir de ne pas y participer à tout moment, et il n'est pas nécessaire de faire des évaluations assignées. Les évaluations simulées sont principalement utilisées comme informations pour le NAT pour évaluer les futurs membres du NAT, bien que les détails présentés dans les évaluations simulées puissent être ajoutées à l'occasion aux commentaires des candidats ou servir de bris d'égalité si le NAT ne peut pas se décider par lui-même.

Reviewer 2: ```suggestion
Les BN non probatory peuvent être assignés au hasard pour faire des évaluations simulées sur les requêtes des BN aux côtés de membres du NAT. Ils peuvent choisir de ne pas y participer à tout moment, et il n'est pas nécessaire de faire des évaluations assignées. Les évaluations simulées sont principalement utilisées comme informations pour le NAT pour évaluer les futurs membres du NAT, bien que les détails présentés dans les évaluations simulées puissent être ajoutées à l'occasion aux commentaires des candidats ou servir de bris d'égalité si le NAT ne peut pas se décider par lui-même.
```

> Les Full BN peuvent être assignés au hasard pour faire des évaluations simulées sur les requêtes des BN aux côtés des vrais du NAT. Ils peuvent choisir de ne pas y participer à tout moment, et il n'est pas nécessaire de faire des évaluations assignées. Les évaluations simulées sont principalement utilisées comme informations pour le NAT pour évaluer les futurs membres du NAT, bien que les détails présentés dans les évaluations simulées puissent être ajoutées à l'occasion aux commentaires des candidats ou servir de bris d'égalité si le NAT ne peut pas se décider par lui-même.

Reviewer 2: ```suggestion
Les BN non probatory peuvent être assignés au hasard pour faire des évaluations simulées sur les requêtes des BN aux côtés de membres du NAT. Ils peuvent choisir de ne pas y participer à tout moment, et il n'est pas nécessaire de faire des évaluations assignées. Les évaluations simulées sont principalement utilisées comme informations pour le NAT pour évaluer les futurs membres du NAT, bien que les détails présentés dans les évaluations simulées puissent être ajoutées à l'occasion aux commentaires des candidats ou servir de bris d'égalité si le NAT ne peut pas se décider par lui-même.
```

- Le [Mapset Verifier](https://osu.ppy.sh/community/forums/topics/943895) : Un outil créé par ([Naxess](https://osu.ppy.sh/users/8129817)), un membre du NAT. Cet outil permet de vérifier un large éventail de problèmes rendant une beatmap non-classable, ainsi qu'une fonctionnalité de capture instantanée pour facilement comparer les versions d'une map, ce qui est utile lors des revérifications. **Cela ne doit pas remplacer la vérification d'une beatmap par vous même**, mais plutôt être utilisé comme un outil de double-vérification au cas-où vous auriez loupé quelque chose.

## Les capacités des Beatmap Nominators

Les nominations et les resets de nominations démontrent les efforts d'un BN lors de la vérifications des maps, indiquant la progression d'une map à travers le processus de classement. En tant que BN vous avez accès à ces outils et vous devez les utiliser à bon escient.

### Nominer

> ![Bouton de nomination](img/nominate-FR.png "Merci à Zelq pour la photo")

Reviewer 2: ```suggestion
![Bouton de nomination](img/nominate-fr.png "Merci à Zelq pour la photo")
```

**Ce bouton vous permet de nominer des maps.** Pour nominer une beatmap, cette dernière doit avoir au moins 5 Hype et aucun problèmes ou de suggestions non résolu. Assurez vous que vous êtes satisfait de toutes les difficultés, de même pour les autres modes de jeu, avant d'utiliser ce bouton. Si la beatmap a fait l'objet d'un veto par le passé, vous devez faire attention à ce que le veto soit retiré avant de faire avancer une beatmap. Quand une beatmap possède 2 nominations, elle sera qualifiée et entrera dans la file d'attente pour le classement.

### Disqualifier ou réinitialiser la nomination

> ![Bouton de disqualification](img/reset_d-FR.png "Merci à Zelq pour la photo")

Reviewer 2: ```suggestion
![Bouton de disqualification](img/reset_d-fr.png "Merci à Zelq pour la photo")
```

> ![Bouton de réinitialisation de la nomination](img/reset_q-FR.png "Merci à Zelq pour la photo")

Reviewer 2: ```suggestion
![Bouton de réinitialisation de la nomination](img/reset_q-fr.png "Merci à Zelq pour la photo")
```
